src/views/mfa-verify/PushForm.js

```diff
@@ -50,6 +50,19 @@ function (Okta, CookieUtil, Util, NumberChallengeView) {
         }
       );
 
+      this.listenTo(this.options.appState, 'change:isMfaRejectedDueToOutdatedApp',
+        function (state, isMfaRejectedDueToOutdatedApp) {
+          this.setSubmitState(isMfaRejectedDueToOutdatedApp);
+          if (isMfaRejectedDueToOutdatedApp) {
+            if (this.options.appState.get('factor').profile.platform === 'IOS') {
+              this.showError(Okta.loc('oktaverify.rejected.upgradeRequired.ios', 'login'));
```

Comment:
i wonder if we can just pass the argument to i18n property, rather than using two different property variables for the same sentence.

Reply:
I thought so initially but there is a subtle difference. 

'on the App Store' and 'on Google Play'

Reply:
you can pass `the App Store` and `Google Play` right ?

Reply:
'the' in other languages might not be 'the' since it unlike 'App Store'

Reply:
Interesting. Didn't think that way.